guides/doc-Administering_with_Ansible/master.adoc

```diff
@@ -1,5 +1,5 @@
 include::common/attributes.adoc[]
-:imagesdir: common/images
+:imagesdir: images
```

Comment:
This also consolidates where images should be find - they are always copied by make into the BUILD_DIR/images so no longer we will confuse common and non-common images. They are all copied to the same destination, of course locals overriding common.